BrownField/Auto-assessment/scripts/Main.ps1

```diff
@@ -8,23 +8,23 @@
 function Main {
     try {
         
-        $tenantId = ""
-        $subscriptionId = ""
+        $tenantId = "27eda52d-06a5-4e9f-bd76-1a062e47aba0"
```

Comment:
@Mahesh-MSFT, it might be better to remove hardcoded tenantId and subscriptionId. What do you think?

Reply:
Done.